mlrun/config.py

```diff
@@ -465,9 +465,18 @@
     },
     "default_workflow_runner_name": "workflow-runner-{}",
     "sidecar": {
-        "log_collector": {
+        "logs_collector": {
             "address": "",
-            "enabled": False,
+            # log collection mode can be one of: "sidecar", "legacy", "best-effort"
+            # "sidecar" - use the sidecar to collect logs
+            # "legacy" - use the legacy log collection method (logs are collected straight from the pod)
+            # "best-effort" - use the sidecar, but if for some reason it's not available use the legacy method
+            # note that this mode also effects the log querying method as well, meaning if the mode is "best-effort"
+            # the log query will try to use the sidecar first and if it's not available it will use the legacy method
+            "mode": "legacy",
```

Comment:
TODO: open PR to v3io/helm-charts to pass the correct env var/vars

Reply:
Also, the base GRPC client used the previous `enabled` so you need to make it use the `mode` now.